### ISSUE TITLE
"Add your plugin" button in the plugin page

### ISSUE THREAD

**Issue by metcoder95** [fastify/website#134]

**Comment by metcoder95** [comment]
I imagine the idea is to create  a section that links to how to create a PR to add a plugin into the Ecosystem section, right?

**Comment by Eomm** [comment]
Yeah, we have already the instruction to add an org, but we don't have info about adding a plugin

**Comment by metcoder95** [comment]
Nice 🚀 
Do you believe we should reuse the same format as per [How To Be Featured?](https://fastify.dev/organisations#how-to-be-featured-here) or there's something else you would like to try?

Mind wandering, I imagine that in the `Ecosystem` section, we have a small section that says something like:

`Did you make a plugin and want it to be listed in the Community section, see <how to do it>`

**Comment by Puppo** [comment]
@Eomm @metcoder95 can I help on this one?

**Comment by Eomm** [comment]
Go for it!
